sdk/python/kfmd/metadata.py

```diff
@@ -46,6 +46,10 @@ def __init__(self,
       labels {object} Optional key/value string pairs to label the workspace.
     """
     # TODO(zhenghuiwang): check each field's type and whether set.
+    if backend_url_prefix is None and type(backend_url_prefix) != str:
+          raise ValueError("'backend_url_prefix' must be set and in string type.")
+    if name is None and type(name) != str:
```

Comment:
it should be `if name is None or type(name) != str`, not `and`.

Same below.

Reply:
Yes. I will change it now.

Reply:
Thanks for your review!